test/libsolidity/semanticTests/externalContracts/deposit_contract.sol

```diff
@@ -179,7 +179,7 @@ contract DepositContract is IDepositContract, ERC165 {
 // ----
 // constructor()
 // gas irOptimized: 1826340
-// gas legacy: 2558004
+// gas legacy: 2575294
```

Comment:
Why did the cost change here?

Reply:
Hmm.. I think this is related to that "threshold".. I guess I removed once the gas costs and let it again update with isoltest. I will verify that all gas costs where not touched in this PR.

Reply:
Ok, that was the only one. But why do we have these "thresholds"?

Reply:
I think it's mainly watching for significant differences between versions because that usually suggest a deeper problem

Reply:
Looks like this change is no longer there. Marking as resolved.